pegasus/sites.v3/hourofcode.com/public/events/2016/index.haml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Yes, it's an archive. We show events from past years on hourofcode.com/events/201X and the latest events on hourofcode.com/events/all.